File: angular-seed/src/app/models/temperature.ts
export class Temperature {
    private idParametro: Number;
    private Clientes_idClientes: Number;
    private valor: Number;
    private fecha: Number;
    private Arduino_idArduino: Number;

    constructor(idParametro:Number,Clientes_idClientes:Number,valor: Number, fecha: Number,Arduino_idArduino: Number) {
        this.idParametro = idParametro;
        this.Clientes_idClientes = Clientes_idClientes;
        this.valor = valor;
        this.fecha = fecha;
        this.Arduino_idArduino = 1;
    }
}

File: src/main/java/com/eci/cosw/springbootsecureapi/model/Temperature.java
package com.eci.cosw.springbootsecureapi.model;

import java.util.Date;

public class Temperature {
    private int idParametro ;
    private int Clientes_idClientes;
    private Float valor;
    private Float fecha;

    public int getIdParametro() {
        return idParametro;
    }

    public void setIdParametro(int idParametro) {
        this.idParametro = idParametro;
    }

    public int getClientes_idClientes() {
        return Clientes_idClientes;
    }

    public void setClientes_idClientes(int clientes_idClientes) {
        Clientes_idClientes = clientes_idClientes;
    }

    public Float getValor() {
        return valor;
    }

    public void setValor(Float valor) {
        this.valor = valor;
    }

    public Float getFecha() {
        return fecha;
    }

    public void setFecha(Float fecha) {
        this.fecha = fecha;
    }

    public Temperature(int idParametro, int clientes_idClientes, float i, float i1){
        this.idParametro=(idParametro);
        this.valor=(i);
        this.fecha=(i1);
        this.Clientes_idClientes=(Clientes_idClientes);
    }

    public Temperature(int idParametro,int Clientes_idClientes, Float valor, Float fecha){
        this.setIdParametro(idParametro);
        this.setValor(valor);
        this.setFecha(fecha);
        this.setClientes_idClientes(Clientes_idClientes);

    }


}
